examples/platform/silabs/MatterConfig.h

```diff
@@ -22,14 +22,24 @@
 #include <platform/CHIPDeviceLayer.h>
 #include <platform/KeyValueStoreManager.h>
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+int32_t wfx_wifi_rsi_init(void);
```

Comment:
No this need to be removed and place somewhere else. This is a generic file. That's not the place to put some platform specific C function declaration

Reply:
done in Latest commit